docs/source/zero-code-change.rst

```diff
@@ -7,19 +7,29 @@ to import equivalent cuML functionality, simply invoke your existing,
 unaltered Python script as follows, and cuML will accelerate as much of the
 code as possible with NVIDIA GPUs, falling back to CPU where necessary:
 
-.. code-block:: terminal
+.. code-block::
+
     python -m cuml.accel unchanged_script.py
 
 The same functionality is available in Jupyter notebooks using the
 following magic at the beginning of the notebook (before other imports):
 
-.. code-bloc:: jupyter
+.. code-block::
+
    %load_ext cuml.accel
    import sklearn
 
 **``cuml.accel`` is currently a beta feature and will continue to improve over
 time.**
 
+.. toctree::
```

Comment:
No need to revert this change. This one should definitely be there. :grin: Thanks!